chargeOclock/devTemp/devClientsViewController.swift

```diff
@@ -47,4 +51,32 @@ class devClientsViewController: UITableViewController {
 		}
 	}
 
+	func plusTapped() {
+
+		let alert = UIAlertController(title: "Add Client 👔", message: "Please, enter client name 😉", preferredStyle: .alert)
+
+		alert.addTextField { textField in
+			textField.placeholder = "Client name"
+		}
+
+		alert.addAction(UIAlertAction(title: "Cancel", style: .cancel, handler: nil ))
+
+		alert.addAction(UIAlertAction(title: "Create", style: .default) { [weak alert] (_) in
```

Comment:
Regarding `[weak alert]` I wasn't careful enough. I thought that the alert controller was owned by self. But it's not, so I removed it.